examples/timestamping/backend/src/proto/timestamping.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.examples.timestamping;
+
+import "helpers.proto";
+import "google/protobuf/timestamp.proto";
+
+message Timestamp {
+  exonum.Hash content_hash = 1;
+  string metadata = 2;
+}
+
+message TimestampEntry {
+  Timestamp timestamp = 1;
+  exonum.Hash tx_hash = 2;
+  google.protobuf.Timestamp time = 3;
```

Comment:
I wonder, is it okay to use `google.protobuf.Timestamp` directly in Protobufs? Maybe it would be better to wrap it into `exonum.DateTime` or something so that it's more clear that it will be mapped onto some Exonum-specific timestamp representation.